qavgpool benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29274 qadaptive_avgpool2d benchmarking
* **#29268 qavgpool benchmarking**
* #29257 Adding short tests to qmaxpool benchmarking

Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
qavgpool benchmarking

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qpool_test.py:
```diff
@@ -80,8 +74,25 @@ def forward(self):
+
+op_bench.generate_pt_test(qmaxpool2d_short_configs, QAvgPool2dBenchmark)
 op_bench.generate_pt_test(qmaxpool2d_short_configs, QMaxPool2dBenchmark)
 op_bench.generate_pt_test(qmaxpool2d_long_configs, QMaxPool2dBenchmark)
```
Concatenate configs such as: 
op_bench.generate_pt_test(qmaxpool2d_short_configs + qmaxpool2d_long_configs, QAvgPool2dBenchmark)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

mingzhe09088 reviewed (approved):
LGTM!

Commit:
Rebasing on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)

Commit:
Update on "qavgpool benchmarking"


Differential Revision: [D18342589](https://our.internmc.facebook.com/intern/diff/D18342589)
